### PR TITLE
CI 1.53 release notes

### PR DESCRIPTION
**Pull request by nofarblue**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: CI 1.53 release notes
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: release-notes/continuous-integration.md [hunk 1]
```diff
@@ -20,8 +20,32 @@ These release notes describe recent changes to Harness Continuous Integration.
...
 
+
+
 ## October 2024
 
+### Version 1.53
+<!-- 2024-11-04 -->
+
+#### New features and enhancements
+- Self-Hosted Cache Intelligence and Docker Layer Caching Enhancements - This release introduces enhancements for self-hosted builds, allowing seamless configuration of S3-compatible caching with AWS or GCP connectors using OIDC for authentication These options are behind the feature flags `CI_ENABLE_DLC_SELF_HOSTED` (for Docker layer caching) and `CI_ENABLE_CACHE_INTEL_SELF_HOSTED` (for Cache Intelligence). Contact [Harness Support](mailto:support@harness.io) to enable them.
```
Comment (anurag-harness) [inline release-notes/continuous-integration.md:31]: Should we mention that OIDC for auth is also behind FFs? PL_GCP_OIDC_AUTHENTICATION for GCP and CDS_AWS_OIDC_AUTHENTICATION for AWS

**Review by Sapa96 — APPROVED** [review]

File: release-notes/continuous-integration.md [hunk 2]
```diff
@@ -20,8 +20,34 @@ These release notes describe recent changes to Harness Continuous Integration.
...
+- OIDC, previously available only for Harness Cloud, is now supported for self-hosted builds running on Kubernetes, enhancing security and simplifying authentication. OIDC is currently behind feature flags `PL_GCP_OIDC_AUTHENTICATION` for GCP and `CDS_AWS_OIDC_AUTHENTICATION` for AWS
+
+To enable feature flags, please contact [Harness Support](mailto:support@harness.io). 
+
+#### Fixed issues
+- Storing secrets in custom secret managers is now supported for cache intelligence in the self-hosted flow. (CI-14719, ZD-71881)
+- Fixed an issue where bitbucket tag builds with tags containing slashes were causing errors in execution due to variable codebase.commitSha returning null. Harness now correctly supports tags with slashes for bitbucket and git builds, ensuring SHA values are properly referenced. (CI-14706, ZD-70972)
```
Comment (hemanthmantri) [inline release-notes/continuous-integration.md:38]: <+codebase.commitSha>

File: release-notes/continuous-integration.md [hunk 3]
```diff
@@ -20,8 +20,34 @@ These release notes describe recent changes to Harness Continuous Integration.
...
+- Addressed an issue where pipelines failed at the clone codebase step on Windows infrastructure when using the GitHub SSH connector. (CI-14592, ZD-70570, ZD-71715)
```
Comment (hemanthmantri) [inline release-notes/continuous-integration.md:39]: This is with LFS enabled right? Should me mention that? cc: @devkimittal

**Review by dewan-ahmed — APPROVED** [review]
LGTM

**Pull request merged**